onnxruntime/contrib_ops/cpu/quantization/qlinear_where.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "qlinear_util.h"
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include "core/util/math_cpuonly.h"
+#include "core/providers/cpu/tensor/where_op.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+class QLinearWhere final : public OpKernel {
+ public:
+  QLinearWhere(const OpKernelInfo& info);
+
+  Status Compute(OpKernelContext* context) const override;
+
+ private:
+  const size_t expected_input_count = 9;
```

Comment:
>onst size_t expected_input_count = 9; [](http://example.com/codeflow?start=3&length=37)

nit: constexpr. And please follow the naming style: [https://google.github.io/styleguide/cppguide.html#Constant_Names](https://google.github.io/styleguide/cppguide.html#Constant_Names)